code/modules/mob/living/carbon/alien/humanoid/combat.dm

```diff
@@ -35,7 +35,7 @@
 
 	return "slashes at"
 
-/mob/living/carbon/alien/humanoid/get_unarmed_damage(mob/living/target)
+/mob/living/carbon/alien/humanoid/get_unarmed_damage(var/atom/target)
```

Comment:
why these changes to this and the other get_unarmed_damages?

Reply:
@DeityLink pls respond

Reply:
>>get_unarmed_damage() procs now take atoms instead of mobs as arg so they can be used against non-mobs.

I missed it, he informed me of this, and I forgot to remove this, sorry.